contracts/abstracted_account.algo.ts

```diff
@@ -8,15 +8,16 @@ export class AbstractedAccount extends Contract {
   admin = GlobalStateKey<Address>();
 
   /**
-   * The apps that are authorized to send itxns from the abstracted account
-   * The box map values aren't actually used and are always empty
+   * The apps and addresses that are authorized to send itxns from the abstracted account,
+   * The key is the appID + address, the value (referred to as `end`)
+   * is the timestamp when the permission expires for the address to call the app for your account.
    */
-  plugins = BoxMap<Application, StaticArray<byte, 0>>({ prefix: 'p' });
+  plugins = BoxMap<bytes, uint64>({ prefix: 'p' });
```

Comment:
Instead of using bytes here, I'd prefer to see `{ application: Application, address: Address }`. Probably want to make that a type and use it every else the key is used

Reply:
i was unaware we could do that, cool!

Reply:
Yep! Under the hood it's just a tuple, which in this case is the same as `itob(app) + address` under the hood, but makes the source easier to read and type safe

Reply:
nice, definitely makes things easier